### ISSUE TITLE
Query params are ignored when using cache

### ISSUE THREAD

**Issue by krzychukula** [Schibsted-Tech-Polska/good-guy-http#2]
When using something like:

```
var options = {
            url: "/something",
            qs: {
                queryParam1: 'some',
                queryParam2: 'values'
            }
        };
request(options)
```

Current implementation seem to ignore query params, at least to my understanding: https://github.com/Schibsted-Tech-Polska/good-guy-http/blob/master/lib/caching/request-key.js

**Comment by jakubwasilewski** [comment]
Yeah, your understanding is correct - I missed that option. Will be fixed for next release, and I'll check out if there are any other `request` options which would influence the final URL.

**Comment by jakubwasilewski** [comment]
Should now be fixed by https://github.com/Schibsted-Tech-Polska/good-guy-http/commit/ec0aee3a46cf0b50bf2de334c798f3c67b6a7d0a. We use the same logic `request` uses internally for the `qs` parameter, so the caching keys should now be correct for all the different ways to use it.

**Comment by jakubwasilewski** [comment]
Fix included in 1.4.2 and 1.5.0.
